gradle/dependencies.gradle

```diff
@@ -6,10 +6,10 @@ buildscript {
   ext.layoutLibPrebuiltSha = 'fa3aa65'
 
   ext.versions = [
-    kotlin: '1.6.10',
+    kotlin: '1.7.0',
```

Comment:
Note: Merging this PR will bump everyone using Paparazzi to suddenly use newer Kotlin. So it's not "support Kotlin 1.7.0", but "require Kotlin 1.7.0"

https://github.com/cashapp/paparazzi/blob/549defcd2591b40aebd011ac0a7eecd74c2ea489/paparazzi/paparazzi/build.gradle#L46

Reply:
Yes,  if you updates the kotlin and compose version in project it will broke paparazzi.
Thanks for comment, I'll update PR title and description to provide more context.

Reply:
Description and title for PR updated.
Thanks for comment!